content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md

```diff
@@ -0,0 +1,124 @@
+---
+reviewers:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+title: Debug Init Containers
+content_type: task
+weight: 40
+---
+
+<!-- overview -->
+
+Esta página mostra como investigar problemas relacionados à execução de
+inicialização de Contêineres. As linhas de comando de exemplo abaixo 
+referem-se ao pod como `<pod-name>` e a inicialização de contêineres como `<init-container-1>` e
```

Comment:
"e a inicialização de contêineres" should be "e aos contêineres de inicialização".

As @sftim pointed out, "Init Containers" refer to a specific kind of container in Kubernetes tasked with initializing a pod.